src/api/forms/service.js

```diff
@@ -66,11 +70,14 @@ export async function createForm(metadata, author) {
 
   // Create the metadata document
   const { insertedId: _id } = await formMetadata.create(document)
+  const createdForm = mapForm({ ...document, _id })
+  logger.info(`Form ${createdForm.id} created for form "${metadata.title}"`)
```

Comment:
I think we tend to use single quotes? 🤷‍♂️

```suggestion
  logger.info(`Form ${createdForm.id} created for form '${metadata.title}'`)
```